integration_test/gce/gce_testing.go

```diff
@@ -1151,12 +1114,21 @@ func attemptCreateInstance(ctx context.Context, logger *log.Logger, options VMOp
 	}
 
 	imgProject := options.ImageProject
+
```

Comment:
I wish I could put this comment on line 1850, but github won't let me do that.

This PR is changing the meaning of VMOptions.Platform, VMOptions.Image, and VMOptions.ImageProject and the comments should be updated accordingly. [sidenote: the "Required" comment on VMOptions.Platform is incorrect. It is required that VMOptions.Platform OR VMOptions.Image are specified though.] Also callsites that use these fields should be updated. Now I'm not sure exactly what you want the new fields to look like and how they should relate to each other. Should everybody cram everything into options.Platform, which then parses it back out? Or should callsites that want to deal with the new format of the PLATFORMS env variable do the parsing, populating the appropriate fields of VMOptions like they do today? (except VMOptions.ImageProject would be required now).

Basically, VMOptions is the interface to this function and I want the interface to be self-consistent and documented.

Reply:
c+p from another comment: "Spoken offline, and agree that the long term solution is to replace instances of platform with imageSpec and updated logic. A followup PR will do a sweeping refactor along these lines."